nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -29,7 +29,7 @@
 
 - Convenience options for `amdgpu`, the open source driver for Radeon cards, are now available under [`hardware.amdgpu`](#opt-hardware.amdgpu.initrd.enable).
 
-- [AMDVLK](https://github.com/GPUOpen-Drivers/AMDVLK), AMD's open source Vulkan driver, is now available to be configured under the [`hardware.amdgpu.amdvlk`](#opt-hardware.amdgpu.amdvlk.enable) option.
+- [AMDVLK](https://github.com/GPUOpen-Drivers/AMDVLK), AMD's open source Vulkan driver, is now available to be configured under the {option}`hardware.amdgpu.amdvlk` option.
```

Comment:
IIRC you’re meant to use the redirects file to redirect these to the release notes where they were removed or something. Not sure exactly.

Reply:
This looks ok to me: I'm not aware of another method.